tests/DebugCodeLens/DebugCodeLensProvider.test.ts

```diff
@@ -153,11 +153,19 @@ describe('DebugCodeLensProvider', () => {
       expect(testResultProvider.getResults).toBeCalledWith(document.fileName)
     })
 
-    it('should not show DebugCodeLenses for successful test results', () => {
+    function expectNoDebugCodeLensesForTestStatus(status: TestReconciliationState) {
       const sut = new DebugCodeLensProvider(testResultProvider, true)
-      getResults.mockReturnValueOnce([{ status: TestReconciliationState.KnownSuccess }])
+      getResults.mockReturnValueOnce([{ status }])
 
       expect(sut.provideCodeLenses(document, token)).toEqual([])
+    }
+
+    it('should not show DebugCodeLenses for successful test results', () => {
+      expectNoDebugCodeLensesForTestStatus(TestReconciliationState.KnownSuccess)
+    })
+
+    it('should not show DebugCodeLenses for skipped test results', () => {
+      expectNoDebugCodeLensesForTestStatus(TestReconciliationState.KnownSkip)
```

Comment:
This isn't an egrarious case - but I'd be wary of making functions out of test code unless it's being used 3+ times (or is a lot of copy pasta) mainly so that you can read the full test each time and know every step based on the assertion

Not a problem here though

Reply:
Thanks for the feedback @orta. I'm grateful for the guidance.